Src/Plugins/Tests.Security/TestData/ReviewPotentiallySensitiveData/ExpectedOutputs/SEC102_003.Url.sarif

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
something is wrong. we should see results in the SARIF

Reply:
Wrong extension on test file! Good catch, thank you.